### ISSUE TITLE
Error with the function automatic messages[Bug]: 

### ISSUE THREAD

**Issue by Carlos-EstebanM** [nvdaes/controlUsageAssistant#4]
### Summary

For some application areas, like a spreadsheet in Excel or a presentation in Power Point, the addon has specific messages. For example, "Use the arrow keys to move between spreadsheet cells". When the feature automatic messages are enabled, in these areas NVDA say general messages.

### To reproduce

1. Enable automatic messages by speech or Braille.
2. Open a new spreadsheet in Excel.

### Expected behavior

After of read the cell (in this case, a1), the message of the addon for this area is: Use the arrow keys to move between spreadsheet cells. However, NVDA say a general message. In this case, Press Control+Alt+arrow keys to other table cells. Pressing left or right arrow keys will navigate between columns, and up and down arrow keys will navigate by row.
The commands for table navigation don't are aplicable in Excel. For this reason, is necessary that the automatic messages say the message for this application area. This also is aplicable for other areas with specific messages, like presentations in PowerPoint, the home screen in Windows 8 or the audio editor GoldWave.

### Relevant log output

```shell
ERROR - eventHandler.executeEvent (10:56:12.697) - MainThread (10772):
error executing event: gainFocus on <NVDAObjects.UIA.UIA object at 0x00FF9E90> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 284, in executeEvent
  File "eventHandler.pyc", line 98, in __init__
  File "eventHandler.pyc", line 107, in next
  File "C:\Users\carlo\AppData\Roaming\nvda\addons\controlUsageAssistant\globalPlugins\controlUsageAssistant\__init__.py", line 170, in event_gainFocus
    if obj.role == self.prevObj.role:
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\window\excel.pyc", line 1345, in _get_role
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\window\excel.pyc", line 1420, in _get_states
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\window\excel.pyc", line 1227, in _get_excelCellInfo
  File "comtypes\client\lazybind.pyc", line 60, in __call__
  File "comtypes\automation.pyc", line 729, in _invoke
  File "monkeyPatches\comtypesMonkeyPatches.pyc", line 32, in __call__
_ctypes.COMError: (-2146827864, None, (None, None, None, 0, None))
```

### Version

2022.03.27-dev

### System configuration

NVDA 2022.1 beta3

**Comment by nvdaes** [comment]
Thanks for reporting. I"ll try to fix this ASAP.

**Comment by nvdaes** [comment]
Fixed in PR #5 
@Carlos-EstebanM , if you still find problems with this, please reopen this issue or create a new one.
I'll update the documentation and will send a new dev version to addonFiles.
